### PR TITLE
some minor cleanups

### PR DESCRIPTION
**Pull request by girtsf**
* Suppress warnings about conflicting library names in platformio.ini by
  explicitly picking the libraries by id that we want.
* fix unused static function warning by making it not static ;)
* hopefully fix the broken platformio dependency detection for fsm that
  shows up in the CI builds?

Tested: `pio run` builds.

### CODE REVIEW

**Comment by girtsf** [comment]
gotta start somewhere

**Comment by geeksville** [comment]
Super awesome!  I'm on a plane flight today but will merge tomorrow and
bless you as a contributor for GitHub.

(Sent from a phone - please ignore typos)

**Comment by geeksville** [comment]
ooh yah - I'm so excited that this might fix https://github.com/geeksville/Meshtastic-esp32/blob/master/.github/workflows/main.yml CI builds.

File: platformio.ini [hunk 1]
```diff
@@ -21,8 +21,8 @@ framework = arduino
 board_build.partitions = partition-table.csv
 
 ; note: we add src to our include search path so that lmic_project_config can override
-; FIXME fix dependencies on arduino-fsm
-build_flags = -Wall -Wextra -Wno-missing-field-initializers -I.pio/libdeps/esp32/arduino-fsm -Isrc -Os -Wl,-Map,.pio/build/esp32/output.map -DAXP_DEBUG_PORT=Serial 
+; FIXME: fix lib/BluetoothOTA dependency back on src/ so we can remove -Isrc
```
Comment (geeksville) [inline platformio.ini]: yay!

File: platformio.ini [hunk 2]
```diff
@@ -55,14 +55,16 @@ debug_tool = jlink
...
   https://github.com/geeksville/RadioHead.git
-  TinyGPSPlus
+  1655 ; TinyGPSPlus
   https://github.com/geeksville/esp8266-oled-ssd1306.git ; ESP8266_SSD1306 
   AXP202X_Library
   SPI
-  OneButton
-  CRC32 ; explicitly needed because dependency is missing in the ble ota update lib
+  1260 ; OneButton
+  1202 ; CRC32, explicitly needed because dependency is missing in the ble ota update lib
```
Comment (geeksville) [inline platformio.ini]: just curious - why did you need to use the numeric versions here?

Comment (girtsf) [reply platformio.ini]: There are multiple libraries with the same name. If you don't specify one by number, it spits out these warnings:

image

(I wish they had a nicer syntax, e.g., `OneButton#1260` or something)

**Comment by geeksville** [comment]
ah - thanks!
